### PR TITLE
Use 5.7.3 hip version

### PR DESCRIPTION
**Pull request by tpadioleau**

### CODE REVIEW

File: docker/latest/Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN chmod +x /bin/bash_run \
     wget \
  && if [ "xhip" = "x${BACKEND}" ] \
   ; then echo 'deb [arch=amd64] https://repo.radeon.com/amdgpu/latest/ubuntu jammy main' > /etc/apt/sources.list.d/amdgpu.list \
-      && echo 'deb [arch=amd64] https://repo.radeon.com/rocm/apt/latest jammy main' > /etc/apt/sources.list.d/rocm.list \
+      && echo 'deb [arch=amd64] https://repo.radeon.com/rocm/apt/5.7.3 jammy main' > /etc/apt/sources.list.d/rocm.list \
```
Comment (jbigot) [inline docker/latest/Dockerfile:27]: if the goal of this is to test the latest versions available, shouldn't we keep latest here?

**Review by jbigot — APPROVED** [review]
sounds great for the oldest one, not sure for the latest

File: docker/latest/Dockerfile (hunk 1 above)
Comment (tpadioleau) [reply docker/latest/Dockerfile:27]: I could not make it work with latest, I think it shall be discussed with Ginkgo dev.

Comment (jbigot) [reply docker/latest/Dockerfile:27]: Sure, but we should maybe create an issue to switch back to lastest in the jammy build (actually, we should maybe switch to mantic for "latest")

Comment (tpadioleau) [reply docker/latest/Dockerfile:27]: I thought it was latest LTS ?

Comment (jbigot) [reply docker/latest/Dockerfile:27]: From my point of view, the best is to test with the oldest version we claim to support and the latest that's easy to install. Then it's just a question of what we define as "easy to install"

**Pull request merged**